lineapy/graph_reader/session_artifacts.py

```diff
@@ -291,7 +299,7 @@ def _slice_session_artifacts(self) -> None:
         self.artifact_nodecollections = list()
         for node_id, n in self.node_context.items():
             if n.assigned_artifact is not None and node_id in [
-                art._node_id for art in self.artifact_list
+                art.node_id for art in self.artifact_list
```

Comment:
purposeful change? Looks like it is still `_node_id` in LineaArtifact class unless art is not a LineaArtifact object here?

Reply:
Clarified with @andycui97 in person